src/layer/water.js

```diff
@@ -206,7 +206,7 @@ export const waterPointLabel = {
   filter: ["all", ["==", ["geometry-type"], "Point"]],
   layout: {
     "text-field": Label.localizedName,
-    "text-font": ["Open Sans Bold Italic"],
+    "text-font": ["Americana-Bold-Italic"],
```

Comment:
Eventually, if gl-native’s local glyph rendering functionality expands beyond CJK, the system may need to match the specified fontstack’s weight and style based on words like “Bold” and “Italic” in the name, but the hyphens would foil that. For now, it doesn’t matter though.

/ref https://github.com/mapbox/mapbox-gl-native/issues/7862#issuecomment-613760767 https://github.com/mapbox/mapbox-gl-native/pull/16253#issuecomment-618730298